### PR TITLE
Calendar Element

### PR DESCRIPTION
**Pull request by MastersAbh**
Calendar template added which detects the Timezone of the user and displays the calendar accordingly. 
I have added a link on the homepage as a rough work. Please give your comments on this.

### CODE REVIEW

**Comment by botanicvelious** [comment]
1) please fix the mainpage template so it isnt a pure link, you can see how it should work at http://python-gsoc.org/deadlines.html
2) please dont commit the project.db
3) from django.urls import path is in urls.py twice

**Comment by MastersAbh** [comment]
#38 Calendar page element -- Made the changes required.

**Comment by botanicvelious** [comment]
your commit actually deletes the project.db now

Also on the homepage it still says "<a href="\calendar">GO TO CALENDAR</a>"

Lastly, it looks like /en/schedule/ was not updated to use your template but you are directing to /calendar/. 

Other then that looks good.

**Comment by botanicvelious** [comment]
Looks like you also left

<p>Interested in volunteering with the Python Software Foundation?  {{ TIME_ZONE }}</p>

in the mentors template.

**Comment by MastersAbh** [comment]
Hey, had some questions.

**Comment by dotgourav** [comment]
> Hey, had some questions.

Just ask the question. People here reside in different timezones and they might not be active at the given moment. So, just ask the question and whenever someone is online, they would answer your question.

**Comment by botanicvelious** [comment]
@MastersAbh what questions?

**Comment by MastersAbh** [comment]
Hey, No questions. Was stuck at a point, that I was able to figure out. 
Anyway, please review the changes. :)

File: gsoc/templates/homepage.html [hunk 1]
```diff
@@ -16,6 +16,9 @@ <h1>Python Summer of Code</h1>
 			</p>
 		</div>
 	</div>
+	<div id="calendar">
+	<a href="\calendar">GO TO CALENDAR</a>
```
Comment (botanicvelious) [inline gsoc/templates/homepage.html]: Why is this still here?

File: gsoc/templates/mentors.html [hunk 2]
```diff
@@ -3,13 +3,11 @@
...
 Mentors
 {% endblock %}
-
 {% block content %}
 	<div class="content">
 		<div class="pure-u-1 pure-u-md-1-1 pure-u-lg-1-1">
 			<h2 class="content-head content-head is-center">Mentors</h2>
-			<p>Interested in volunteering with the Python Software Foundation?</p>
-
+			<p>Interested in volunteering with the Python Software Foundation?  {{ TIME_ZONE }}</p>
```
Comment (botanicvelious) [inline gsoc/templates/mentors.html]: Left over Debug data?

File: gsoc/urls.py [hunk 3]
```diff
@@ -31,7 +33,7 @@
     path('admin/', admin.site.urls),
     url(r'^', include('cms.urls')),
 )
-
+urlpatterns += [path('calendar/', views.calender)]
```
Comment (botanicvelious) [inline gsoc/urls.py]: this still needed?

File: gsoc/urls.py [hunk 4]
```diff
@@ -12,6 +12,8 @@
 from django.views.static import serve
 from django.urls import path
 
+from . import views
```
Comment (botanicvelious) [inline gsoc/urls.py]: unneeded now?

File: gsoc/views.py [hunk 5]
```diff
@@ -0,0 +1,41 @@
+from django.http import HttpResponse
```
Comment (botanicvelious) [inline gsoc/views.py]: Is any of this file needed if you are using a tag?

**Review by botanicvelious — CHANGES_REQUESTED** [review]
- [ ] I see no changes to the schedule template
>python-blogs/gsoc/templates/schedule.html
- [ ] the project.db is still being erased by this pr

- [ ] other review items below as well



>

**Comment by MastersAbh** [comment]
I am just not committing project.db . Is it still deleting it for you?

**Comment by botanicvelious** [comment]
> 
> 
> I am just not committing project.db . Is it still deleting it for you?

https://github.com/python-gsoc/python-blogs/pull/47/files

You can see at the bottom BIN -1.24 MB project.db

**Comment by botanicvelious** [comment]
Looks good just fix the project.db issue and its good to go.

**Review by botanicvelious — APPROVED** [review]

**Review by botanicvelious — CHANGES_REQUESTED** [review]
Don't delete project.db

**Review by botanicvelious — APPROVED** [review]

**Pull request merged**